index.md

```diff
@@ -88,13 +88,18 @@ Make your event and your presentations accessible so that you are prepared for s
 
 Presentations that are accessible to people with disabilities are also **inclusive** to many more audiences. For example, people with different learning styles and people who are not fluent in the language.
 
-Accessible presentations also have additional benefits, such as in these situations:
+Accessible presentations also have additional benefits, such as in this situation:
 
--  A live presentation with visuals is recorded and made available online as an audio podcast. During the presentation you describe the visuals. That provides the visual information to people who are blind or can't see the visuals well during the presentation. People listening to the podcast later will also get the visual information.
--  [CART](#cart){:.termref} provides real-time text of the speaker and other audio. People who are deaf or hard of hearing use CART. CART also helps people who understand text better than spoken language. For example, people whose native language is different. CART output can also be used to develop a transcript.
--  Transcripts online can increase search engine optimization (SEO) for audio and video. [Transcripts also have many other benefits {% include_cached external.html %}](http://www.uiaccess.com/transcripts/transcripts_on_the_web.html).
+-  A live presentation with “slides” is recorded and made available online as an audio podcast. The presenter describe the important visual information in the slides during the presentation.
```

Comment:
Grammar: "the presenters describe" vs "the presenter describes"?

Reply:
yup